common/app/services/newsletters/GroupedNewslettersAgent.scala

```diff
@@ -0,0 +1,57 @@
+package services.newsletters
+
+import com.gu.Box
+import common.GuLogging
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class GroupedNewslettersAgent(newsletterApi: NewsletterApi) extends GuLogging {
+
+  val emptyGroupedNewslettersResponse = GroupedNewslettersResponse(
+    GroupedNewsletterResponse("News roundups", Nil),
```

Comment:
It's a bit of a shame for frontend to maintain any knowledge (outside tests/fixtures) of the categories. Especially as we'll never keep them up to date! Could we just return a completely empty `GroupedNewslettersResponse`?

Reply:
Yeah, I can't see an obvious way around this. I looked at it when I implemented `GroupedNewslettersResponse` a while ago and the problem was checking that the response we got was correct in terms of the fields. The only way I could do that was by specifying them explicitly.

I'll implement an empty response field for now and add a note that it would be nice to not have this. I'll also move it into the `object`/`case class` to keep it together